features/auth-not-authorized.feature

```diff
@@ -0,0 +1,39 @@
+Feature: Check that user is not able to use endpoints without authorization
+
+  Scenario Outline: Requested endpoint should return status 404 when user is not authorized
+    Given api link "<link>"
+    When use given link to make a "<method>" api request
+    Then I should get "<status>"
+
+  Examples:
```

Comment:
it doesn't make sense to check just the endpoint status. This type of test is useless. Think about what are you trying to test, not why are doing it.